harness/tests/determined/common/experimental/test_experiment.py

```diff
@@ -271,7 +217,7 @@ def test_list_checkpoints_errors_on_only_order_by_set(
     expref = make_expref(1)
 
     with pytest.raises(AssertionError):
-        expref.list_checkpoints(sort_by=None, order_by=checkpoint.CheckpointOrderBy.ASC, limit=5)
```

Comment:
Guess it was a bad idea to include `limit` here in the first place -- made the test more brittle. Looks like something we both missed.